### PR TITLE
Feat/sep 18

### PR DESCRIPTION
**Pull request by Huynhtram1504**
## Summary by CodeRabbit

- **New Features**
	- Introduced automated tests for product interactions and To-Do application functionalities.
	- Added a utility function to check for prime numbers and a function to calculate age based on the birth year.
- **Bug Fixes**
	- Enhanced error handling in tests to ensure expected outcomes are verified accurately.

### CODE REVIEW

File: daily-challenges/2024-09/17/huynhtram1504.spec.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    await page.locator("//button[@data-product-id='3']").click({clickCount:3});
+    //Kiểm tra số lượng hàng đúng với mô tả
+    await expect(page.locator("//tbody[@id='cart-items']/tr[1]/td[3]")).toHaveText("2");
+    await expect(page.locator("//tbody[@id='cart-items']/tr[2]/td[3]")).toHaveText("2");
+    await expect(page.locator("//tbody[@id='cart-items']/tr[3]/td[3]")).toHaveText("3");
+
+    //(Nâng cao) Kiểm tra tổng tiền sản phẩm đúng (tổng tiền = tổng (số lượng * đơn giá))
+    let totalMoney = 0;
+    product.forEach(item => totalMoney += item.price * item.quantity);
+    //Retreives all matching rows into an array
```
[the remark this answers is not in the document]
Comment (ththao27) [reply daily-challenges/2024-09/17/huynhtram1504.spec.ts:71]: có thể tham khảo thêm đoạn code bot recommend nhé

**Comment by ththao27** [comment]
please fix comments

File: daily-challenges/2024-09/17/huynhtram1504.spec.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import{test, expect} from "@playwright/test";
+//Javascript
+// Check xem 1 số có phải là số nguyên tố hay không
+function isPrime(num: number): boolean{
+    // Số nhỏ hơn 2 không phải là số nguyên tố
+    if(num < 2){
```
Comment (minhphong306) [inline daily-challenges/2024-09/17/huynhtram1504.spec.ts:6]: Please format code.

**Review by minhphong306 — CHANGES_REQUESTED** [review]
Please fix comments

**Pull request closed without merging**